src/modality-specific-files/magnetic-resonance-imaging-data.md

```diff
@@ -1,5 +1,31 @@
 # Magnetic Resonance Imaging
 
+## Data format
+
+All MRI imaging data MUST be stored using the [NIfTI-1 or NIFTI-2](https://nifti.nimh.nih.gov/)
+file format (.nii). The ANALYZE-7.5 file format (.hdr/.img), despite
+being very similar to NIFTI-1, is not allowed. Imaging data SHOULD be
+converted to the NIfTI format using a tool that provides as much of
+the NIfTI header information (such as orientation and slice timing
+information) as possible.
+
+We RECOMMEND to use [dcm2niix](https://github.com/rordenlab/dcm2niix)
+to convert source DICOM images to NIFTI-2 file format, as it extracts
+large number of BIDS compatible metadata. As alternative, [dicm2nii](https://www.mathworks.com/matlabcentral/fileexchange/42997-xiangruili-dicm2nii) and
+[spm_dicom_convert](https://github.com/neurodebian/spm12/blob/master/spm_dicom_convert.m)
+from [spm package](https://github.com/neurodebian/spm12/tree/master) can be used for conversion.
+
+Multi-volume acquisitions (for example, 2D slices of anatomical MRI or 3D volumes
+of functional MRI) MUST be concatenated into single image (3D or 4D, respectively).
+This can be performed either by [concat_images](https://nipy.org/nibabel/reference/nibabel.funcs.html)
+of Nibabel package.
+
+Due to the important size of MRI imaging data, we RECOMMEND using compressed NIfTI files
+by [gzip](https://www.gzip.org/) algorithm (.nii.gz).
```

Comment:
Here I wanted to stress that compressed files should conserve their original extension and have `.gz` additional extension (in case if gzip can support several ones). But if it obvious, I agree with changes.

Reply:
Good point. 

Pinging @effigies and @sappelhoff to get second opinions.

Can briefly talk about this at the maintainers meeting tomorrow.

Reply:
🤷 I think it's fine to mention `.nii.gz` in brackets here.